### DISCUSSION TITLE
MaxTransferSize for DBBackup

### DISCUSSION THREAD

**Discussion by Nehamnsu** [olahallengren/sql-server-maintenance-solution#445] [Questions & Answers] [answered]
We have TDE Enabled and are on SQL Server 2017. My understanding was that if we set @Compress parameter to "Y" then the backups will get compressed automatically. However, it seems like we need to explicitly set the "MaxTransferSize" to leverage compression. 

Can you please recommend a value for "MaxTransferSize parameter? Our DB is about 890 GB in size and we are on AWS EC2 instance.

**Comment by olahallengren** [comment, marked as answer]
@Nehamnsu, I have added logic in the script so that the required value for MAXTRANSFERSIZE is set automatically. You don't need to do anything other than setting the parameter ```@Compress = 'Y'```.

**Comment by Nehamnsu** [comment]
THank you
